server.py

```diff
@@ -170,7 +170,10 @@ def object_as_dict(obj):
 
 def save_response(bound_logger, survey_response):
     bound_logger.info("Saving response")
+
     invalid = survey_response.get("invalid")
+    if invalid:
+        survey_response.pop("invalid")
```

Comment:
For this and the one in the feedback part.  This needs to be in a try except block as if the invalid key is missing for whatever reason then it'll throw a keyError.  I **think** flask handles it, but it'd be better to explicitly handle it.

Reply:
We should consider changing sdx-collect to pass the invalid property outside of the submitted data.

Reply:
Yea, actually that's a fair point.  I foolishly didn't look at how we were getting this `invalid` variable